ci/docker-publish.yml

```diff
@@ -41,9 +41,9 @@ jobs:
       # https://github.com/sigstore/cosign-installer
       - name: Install cosign
         if: github.event_name != 'pull_request'
-        uses: sigstore/cosign-installer@d6a3abf1bdea83574e28d40543793018b6035605
+        uses: sigstore/cosign-installer@6679bb8590baffcdcab1351fef9c80d39e0fa2c9
```

Comment:
@dlorenc Kindly update this to a release tag commit sha. Like v2.4 `7e0881f8fe90b25e305bbf0309761e9314607e25`.